tcl/org.testeditor.tcl.dsl.tests/src/org/testeditor/tcl/dsl/validation/TclValidatorTest.xtend

```diff
@@ -36,7 +37,10 @@ class TclValidatorTest extends AbstractParserTest {
 		var modelError = parseTcl(tclError.toString, "Test2.tcl")
 
 		// then
-		validator.assertNoIssues(model)
+		validator.validate(model).assertSingleElement => [
```

Comment:
I would have expected maybe one test dedicated to the presence / absence of the missing fixture exception warning, each. It would be nicer, I think, if the pre-existing tests here (which originally had a testing intention separate from this whole FixtureException business) would use fixture methods that actually do declare FixtureException.

I suppose a more basic problem is that this test seems to rely on SWTFixture, and if that makes it significantly harder to modify the tests and/or necessitate adapting SWTFixture, as well, maybe it shouldn't depend on it?